src/schema/feeds.ts

```diff
@@ -1171,6 +1208,61 @@ export const resolvers: IResolvers<any, Context> = traceResolvers({
       }
       return feedResolverV1(source, args, ctx, info);
     },
+    customFeed: async (
+      source,
+      args: ConfiguredFeedArgs,
+      ctx: Context,
+      info,
+    ) => {
+      const feedIdOrSlug = args.feedId;
+
+      const feed = await getFeedByIdentifiers({
+        con: ctx.con,
+        feedIdOrSlug,
+        userId: ctx.userId,
+      });
+      const feedId = feed.id;
+
+      if (
+        args.version >= 2 &&
+        args.ranking === Ranking.POPULARITY &&
+        ctx.userId
+      ) {
+        const feedGenerator = new FeedGenerator(
+          new FeedClient(process.env.POPULAR_FEED),
+          new FeedPreferencesConfigGenerator(
+            {},
+            {
+              includePostTypes: true,
+              includeBlockedSources: true,
+              includeBlockedTags: true,
+              feedId: feedId,
```

Comment:
I think we'll also need the new one `includeBlockedContentCuration` for this feed?

Reply:
For now it should be the same as popular as mentioned in RFC. We can fine tune during testing.

Reply:
Didn't I add it in popular? 😅 (think it's one that goes for all feeds)

Reply:
You are right, missed it in merge.